File: dayTwo/Java/Level4/Task.java
public class Task {
    
   
    public static int addTwoNumbers(int a, int b) {
        return a + b;
    }

  d
    public static String getEven(int number) {
        return (number % 2 == 0) ? "Even" : "Odd";
    }

  
    public static int square(int number) {
        return number * number;
    }

  
    public static double celsiusToFahrenheit(double celsius) {
        return (celsius * 9/5) + 32;
    }

    
    public static int findLargest(int a, int b, int c) {
        return Math.max(a, Math.max(b, c));
    }

    public static void main(String[] args) {
        System.out.println("Addition: " + addTwoNumbers(2, 5));
        System.out.println("Even/Odd: " + getEven(5));
        System.out.println("Square: " + square(4));
        System.out.println("Fahrenheit: " + celsiusToFahrenheit(25));
        System.out.println("Largest: " + findLargest(10, 25, 15));
    }
}

File: dayTwo/JavaScript/Level4/Task.js

function getTwoNumbers(firstNumber, secondNumber) {
  return firstNumber + secondNumber;
}


function getEven(number) {
  return number % 2 === 0 ? "even" : "odd";
}

function getSquare(number) {
  return number * number;
}

function celsiusToFahrenheit(celsius) {
  return (celsius * 9/5) + 32;
}

function getLargest(a, b, c) {
  return max(a, b, c);
}


console.log(getTwoNumbers(4, 6));      
console.log(getEven(4));               
console.log(getSquare(3));             
console.log(celsiusToFahrenheit(30));  
console.log(getLargest(10, 25, 7));    

